catkin_ws/src/state_estimation/launch/state_estimation.launch

```diff
@@ -1,8 +1,8 @@
 <launch>
     <arg name="sim" default="false" />
     <group unless="$(arg sim)">
-		<include file="$(find imu)/launch/imu.launch"/>
-		<include file="$(find dvl)/launch/dvl.launch"/>
+		<include file="$(find sensors)/launch/imu.launch"/>
+		<include file="$(find sensors)/launch/dvl.launch"/>
```

Comment:
I think this should also include the depth sensor launch file? (I see that it wasnt there before but that might have been a bug we hadnt found yet) Correct me if im wrong @MrMondrian